docs/users_guide/ocean/tasks/merry_go_round.md

```diff
@@ -0,0 +1,169 @@
+(ocean-merry-go-round)=
+
+# merry-go-round
+
+The `ocean/merry_go_round` test group induces a convective cell in a horizontal
+domain in order to verify tracer advection.
+
+```{image} images/merry_go_round_section.png
+:align: center
+:width: 500 px
+```
+
+## supported models
+
+These tasks only support MPAS-Ocean.
+
+(ocean-merry-go-round-default)=
+
+## default
+
+### description
+
+For the initial conditions described below, tracer concentration contours match
+the streamlines of the convective cell, such that an accurate tracer advection
+scheme would result in no change in the tracer field in time.
+
+The init step generates the mesh and initial condition for the requested
+resolution.
+
+The forward step runs the model for the requested length of time. Tendencies
+for normal velocity and layer thickness are disabled, such that these fields
+remain fixed at their initial conditions throughout the simulation.
+
+The visualization step produces a plot illustrating the horizontal velocity,
+vertical velocity, simulated `tracer1` concentration, the error in simulated
+tracer concentration at the end of the forward simulation.
+(See above for an example).
+
+### mesh
+The mesh is planar and the resolution is specified by config option
+`convergence:base_resolution`, which defaults to 5 m. The horizontal
```

Comment:
I think it might be confusing for folks that in order to change the resolution for the default task they need to change options in the `convergence` section. I would prefer that we have a new section `merry_go_round_default` that has any options that are used only by the default case (which may be just `resolution`).